README.md

```diff
@@ -28,37 +28,50 @@ This extension should work with the following devices:
 
 ## Installation
 
-### Ubuntu 20.10 or newer (features Inkscape 1.0+ natively)
+### Ubuntu
 
-WARNING: SNAP packages may cause issues.
+WARNING: SNAP packages may cause issues. We use deb file shown later in this section.
 
-Install Inkscape
-* `sudo apt install inkscape`
+<details>
+<summary>Click to get bash commands</summary>
 
-Install requirements
-* `sudo apt install python3-usb`
+```bash
+# Add inkscape dev team's PPA key to APT.
+# This project require minimum of inkscape V1.0+
+# But we want to always keep to latest inkscape version
+sudo add-apt-repository ppa:inkscape.dev/stable
 
-Install inkscape-silhouette
-* https://github.com/fablabnbg/inkscape-silhouette/releases
-  Scroll down to Downloads and click on the *.deb file.
+# Install Inkscape
+sudo apt-get update
+sudo apt install inkscape
 
-### Ubuntu 18.04 or 20.04
+# Install Inkscape with newer version directly from inkscape dev team
+# even if newer than what Ubuntu's package management team is willing
+# to certify at the moment
+sudo apt-get --with-new-pkgs upgrade inkscape
 
-WARNING: SNAP packages may cause issues.
+# Install requirements for usb support
+sudo apt-get install python3-usb
 
-Add Inkscape 1.0+ repo
-* `sudo add-apt-repository ppa:inkscape.dev/stable`
-* `sudo apt-get update`
+# Install requirements for Silhouette Multiple Actions
+sudo apt install python3-wxgtk4.0
 
-Install Inkscape
-* `sudo apt install inkscape`
+# Install all requirements from python package manager
+sudo apt-get install python3-pip
+python3 -m pip install -U pip
+python3 -m pip install -r requirements.txt
+```
+</details>
 
-Install requirements
-* `sudo apt install python3-usb`
+From here, you should have all the required python packages and inkscape version.
+So now we shall install inkscape-silhouette, so scroll down the latest releases and head to the Assets section of releases and click on the *.deb file. You can then use `sudo apt-get install ./*.deb` where `*.deb` is the name of your newly downloaded file.
 
-Install inkscape-silhouette
 * https://github.com/fablabnbg/inkscape-silhouette/releases
-  Scroll down to Downloads and click on the *.deb file.
+
+<details>
+<summary>Maintainers Notes On Missing Dependencies in above instructions</summary>
+Dependencies may change every now and then. (i.e. Inscape 1.4 will require tinycss2). To avoid duplication of effort, try using `pip install -r requirements.txt` to update required. This also helps keeping instructions more platform agnostic (except for GTK and matplotlib which are better managed by the distro)
+</details>
```

Comment:
drop this details... as this installation step is already included in above instructions. that was more reasoning for you to adapt requirements.txt instead, which you did :+1: